src/common/hooks/useOrderLineLocations/useOrderLineLocations.js

```diff
@@ -0,0 +1,54 @@
+import { useMemo } from 'react';
+
+import {
+  useCentralOrderingContext,
+  useHoldingsAndLocations,
+  useLocationsQuery,
+  useReceivingTenantIdsAndLocations,
+} from '@folio/stripes-acq-components';
+import { useStripes } from '@folio/stripes/core';
+
+const DEFAULT_DATA = [];
+
+export const useOrderLineLocations = ({ poLineLocations = DEFAULT_DATA, instanceId }) => {
```

Comment:
It seems to me as a bad naming, because it breaks our hooks naming convention. For `useOrderLineLocations` I expect to provide an PO Line's ID and get locations associated with it. Instead I have to provide another property of locations